pkg/test/istio.io/tasks/security/secure-ingress-sds/curl-helloworld-tls-gateway-gke.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo "INGRESS_HOST ${INGRESS_HOST}"
+export INGRESS_HOST
+
+SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].port}')
+echo "SECURE_INGRESS_PORT ${SECURE_INGRESS_PORT}"
+export SECURE_INGRESS_PORT
+
+echo "curl -v -HHost:helloworld-v1.example.com \
```

Comment:
Add `set -x` at the top of the script. It will have the same effect and will be simpler to maintain.

Reply:
Thanks for the advice. Done.